multimedia/ffmpeg4/Makefile

```diff
@@ -0,0 +1,345 @@
+	--enable-muxer=rtsp,sap,segment,srt \
+	--enable-muxer=stream_segment,truehd \
+	--enable-muxer=wav,webm,webm_chunk \
+	--enable-muxer=webm_dash_manifest,webvtt \
+	--enable-muxer=yuv4mpegpipe \
+	--enable-parser=aac,aac_latm,ac3,dca \
+	--enable-parser=flac,g729,gsm,h263 \
+	--enable-parser=h264,hevc,mjpeg,mpeg4video \
+	--enable-parser=mpegaudio,mpegvideo,opus \
+	--enable-parser=vorbis,vp8,vp9 \
+	--enable-protocol=async,cache,concat,data \
+	--enable-protocol=file,hls,http,https,icecast \
+	--enable-protocol=pipe,rtmp,rtmps,tcp,udp,unix \
+	--enable-bsf=aac_adtstoasc,chomp,dca_core \
+	--enable-bsf=dump_extradata,eac3_core,extract_extradata \
+	--enable-bsf=h264_metadata,h264_mp4toannexb,hevc_mp4toannexb \
+	--enable-bsf=hevc_metadata,hevc_mp4toannexb \
+	--enable-bsf=mjpeg2jpeg,mjpega_dump_header,mlp_parser \
+	--enable-bsf=mp3_header_decompress,mpeg2_metadata \
+	--enable-bsf=mpeg4_unpack_bframes,noise,null \
+	--enable-bsf=remove_extradata \
+	--enable-bsf=vp9_raw_reorder,vp9_superframe \
+	--enable-bsf=vp9_superframe_split \
+	--enable-filter=acopy,adelay,aformat,ainterleave \
+	--enable-filter=ametadata,apad,aresample \
+	--enable-filter=ashowinfo,astreamselect,atrim \
+	--enable-filter=bwdif,channelmap,copy,crop,format \
+	--enable-filter=fps,framerate \
+	--enable-filter=interleave,join,metadata \
+	--enable-filter=movie,null,nullsink \
+	--enable-filter=nullsrc,pad,qp \
+	--enable-filter=realtime,scale,select \
+	--enable-filter=showinfo,streamselect,trim \
+	--enable-filter=volume,w3fdif,yadif  \
+	--disable-bzlib --disable-lzma \
+	--enable-cross-compile --disable-debug \
+	--cross-prefix="$(TARGET_CROSS)" \
+	--arch="$(ARCH)" \
+	--target-os="linux" \
+	--ld="$(TARGET_CC)" \
+	--pkg-config="pkg-config" \
+	$(if $(CONFIG_ffmpeg4_ENC_FLV),--enable-encoder=flv,) \
+	$(if $(CONFIG_ffmpeg4_ENC_MJPEG),--enable-encoder=mjpeg,) \
+	$(if $(CONFIG_ffmpeg4_ENC_MPEG4),--enable-encoder=mpeg4,) \
+	$(if $(CONFIG_ffmpeg4_ENC_PNG),--enable-encoder=png,) \
+	$(if $(CONFIG_ffmpeg4_FEAT_DASH),--enable-demuxer=dash --enable-libxml2,) \
+	$(if $(CONFIG_ffmpeg4_LIB_FDK-AAC),--enable-decoder=libfdk_aac --enable-encoder=libfdk_aac --enable-libfdk-aac --enable-nonfree, --disable-libfdk-aac) \
+	$(if $(CONFIG_ffmpeg4_LIB_LAME),--enable-encoder=libmp3lame --enable-libmp3lame,--disable-libmp3lame) \
+	$(if $(CONFIG_ffmpeg4_LIB_MBEDTLS),--enable-mbedtls --enable-version3,--disable-mbedtls) \
+	$(if $(CONFIG_ffmpeg4_LIB_OPENSSL),--enable-openssl --enable-nonfree,--disable-openssl) \
+	$(if $(CONFIG_ffmpeg4_LIB_LIBOPUS),--enable-decoder=libopus --enable-encoder=libopus --enable-libopus,--disable-libopus) \
+	$(if $(CONFIG_ffmpeg4_LIB_LIBVORBIS),--enable-decoder=libvorbis --enable-encoder=libvorbis --enable-libvorbis,--disable-libvorbis) \
+	$(if $(CONFIG_ffmpeg4_LIB_LIBX264),--enable-encoder=libx264 --enable-encoder=libx264rgb --enable-libx264 --enable-gpl,--disable-libx264) \
+	$(if $(CONFIG_ffmpeg4_OPT_PERFORMANCE),,--enable-small) \
```

Comment:
Trailing \ should be removed on this line

Reply:
No, that would break config args (new line). Can't reproduce any issues on master with Alpine Linux.

Reply:
It doesn't cause any issues as long as the following line is empty (which is the case right now).
However if Makefile is changed and the following line isn't blank it may cause issues.